.dockerignore

```
@@ -26,3 +28,13 @@ shared/tsconfig.build.tsbuildinfo
 
 frontend/.angular
 frontend/cypress*
+
+## VSCode
+
+.vscode
+
+## more unnecessary for production, not needed inside the docker container
+
+docker-compose.yml
+README.md
```

```suggestion
**/README.md
```

---

Or maybe even

```suggestion
**/*.md
```

You'd have to be very, very careful with this, though, as there could be valid use cases for utilizing `.md` files at run-time (although we don't have something like this right now, I think).